release/example/en.wordlist/source/model.ts

```diff
@@ -0,0 +1,11 @@
+import LexicalModelCompiler from "../../../../tools/index";
+
+(new LexicalModelCompiler).compile({
+  format: 'trie-1.0',
+  wordBreaking: {
+    allowedCharacters: { initials: 'abcdefghijklmnopqrstuvwxyz', medials: 'abcdefghijklmnopqrstuvwxyz', finals: 'abcdefghijklmnopqrstuvwxyz' },
```

Comment:
Although I like how compact this notation is, JavaScript's strings may not work well, because there's no explicit way to denote full characters that are greater than one code point (e.g., <X̱> in Northern Haida, two code points in NFC). Also, characters that are outside the BMP will be a pain, and error-prone.

Reply:
So could we support string -- for simple situations or array of strings for others? e.g. `['a','b',...,'X̱']`?

I am unsure on how this is going to be used -- I would happy to have the source format support both but compiled into array of strings for simplicity of consumption.

Reply:
I like this idea!